lib/shared-mongo-connections.js

```diff
@@ -0,0 +1,75 @@
+var mongoUrlParser = require('mongodb-uri');
+var async = require('async');
+var MongoClient = require('mongodb').MongoClient;
+var mongoose = require('mongoose');
+
+var MONGODB_DEFAULT_POOL_SIZE = 20;
+
+/**
+ * Get the url to connect to the admin mongodb specified in the given `config` object.
+ * @param {string} mongoConnectionString a mongodb url that should at least contain the mongodb hosts
+ * @param {object} mongoConf the mongodb configuration
+ * @param {string} mongoConf.admin_auth.user the admin username
+ * @param {string} mongoConf.admin_auth.pass the amdin user password
+ * @param {int} mongoConf.poolSize the poolSize of the mongodb connection. Default to 20.
+ * @return {string} the mongodb admin connection url
+ */
+function getAdminDbUrl(mongoConnectionString, mongoConf) {
```

Comment:
From https://github.com/feedhenry/fh-mbaas/pull/18#pullrequestreview-19732949 .

This should not be an administrative user. It should be a user with only the required permissions to connect to any database.